Sources/Build/llbuild.swift

```diff
@@ -353,6 +353,9 @@ public final class LLBuildManifestGenerator {
         let commands: [Command] = try target.compilePaths().map({ path in
             var args = target.basicArguments()
             args += ["-MD", "-MT", "dependencies", "-MF", path.deps.pathString]
+          #if os(Windows)
+            args += ["-D_MD", "-D_DLL", "-U_DEBUG"]
```

Comment:
This is release mode only.  Can the user build in both debug and release mode?

Reply:
We should look at target triple here.